packages/eas-cli/src/devices/utils/formatDevice.ts

```diff
@@ -20,10 +21,34 @@ export default function formatDevice(device: Device, team?: AppleTeamIdAndName):
 
   if (team) {
     fields.push(
-      ...[
-        { label: 'Apple Team ID', value: team.appleTeamIdentifier },
-        { label: 'Apple Team Name', value: team.appleTeamName ?? 'Unknown' },
-      ]
+      { label: 'Apple Team ID', value: team.appleTeamIdentifier },
+      { label: 'Apple Team Name', value: team.appleTeamName ?? 'Unknown' }
+    );
+  }
+
+  return formatFields(fields);
+}
+
+// TODO: integrate this with `formatDevice`
```

Comment:
Can you do this in this PR? It doesn't seem to be much work.

Reply:
Kept the model part of the device class, if it's available.